tests/src/test/java/alluxio/cli/ConfigurationDocGeneratorTest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli;
+
+import alluxio.Configuration;
+import alluxio.PropertyKey;
+import alluxio.collections.Pair;
+import alluxio.util.io.PathUtils;
+import org.junit.Assert;
```

Comment:
leave an empty line between lien 17 and 18 to separate these two sessions of imports